src/pages/dashboard/monitor/index.tsx

```diff
@@ -0,0 +1,272 @@
+                        />
+                      }
+                      total="28%"
+                      height={128}
+                      lineWidth={2}
+                    />
+                  </Col>
+                  <Col span={8}>
+                    <Pie
+                      animate={false}
+                      color="#5DDECF"
+                      percent={22}
+                      title={
+                        <FormattedMessage
+                          id="dashboardandmonitor.monitor.western-food"
+                          defaultMessage="Western food"
+                        />
+                      }
+                      total="22%"
+                      height={128}
+                      lineWidth={2}
+                    />
+                  </Col>
+                  <Col span={8}>
+                    <Pie
+                      animate={false}
+                      color="#2FC25B"
+                      percent={32}
+                      title={
+                        <FormattedMessage
+                          id="dashboardandmonitor.monitor.hot-pot"
+                          defaultMessage="Hot pot"
+                        />
+                      }
+                      total="32%"
+                      height={128}
+                      lineWidth={2}
+                    />
+                  </Col>
+                </Row>
+              </Card>
+            </Col>
+            <Col xl={6} lg={12} sm={24} xs={24} style={{ marginBottom: 24 }}>
+              <Card
+                title={
+                  <FormattedMessage
+                    id="dashboardandmonitor.monitor.popular-searches"
+                    defaultMessage="Popular Searches"
+                  />
+                }
+                loading={loading}
+                bordered={false}
+                bodyStyle={{ overflow: 'hidden' }}
+              >
+                <TagCloud data={tags || []} height={161} />
+              </Card>
+            </Col>
+            <Col xl={6} lg={12} sm={24} xs={24} style={{ marginBottom: 24 }}>
+              <Card
+                title={
+                  <FormattedMessage
+                    id="dashboardandmonitor.monitor.resource-surplus"
+                    defaultMessage="Resource Surplus"
+                  />
+                }
+                bodyStyle={{ textAlign: 'center', fontSize: 0 }}
+                bordered={false}
+              >
+                <WaterWave
+                  height={161}
+                  title={
+                    <FormattedMessage
+                      id="dashboardandmonitor.monitor.fund-surplus"
+                      defaultMessage="Fund Surplus"
+                    />
+                  }
+                  percent={34}
+                />
+              </Card>
+            </Col>
+          </Row>
+        </React.Fragment>
+      </GridContent>
+    );
+  }
+}
+
+export default Monitor;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)